link/netfilter.go

```diff
@@ -0,0 +1,92 @@
+package link
+
+import (
+	"fmt"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/internal/sys"
+)
+
+const NFIPDefrag NFAttachFlags = 0 // Enable IP packet defragmentation
+
+const (
+	NFINETPrerouting  NFHook = iota // NF_INET_PRE_ROUTING
+	NFINETLocalIn                   // NF_INET_LOCAL_IN
+	NFINETForward                   // NF_INET_FORWARD
+	NFINETLocalOut                  // NF_INET_LOCAL_OUT
+	NFINETPostrouting               // NF_INET_POST_ROUTING
+	NFINETIngress                   // NF_INET_INGRESS
+)
+
+const (
+	ProtoUnspec NFProtoFamily = 0  // NFPROTO_UNSPEC
+	ProtoInet   NFProtoFamily = 1  // NFPROTO_INET
+	ProtoIPv4   NFProtoFamily = 2  // NFPROTO_IPV4
+	ProtoARP    NFProtoFamily = 3  // NFPROTO_ARP
+	ProtoNetDev NFProtoFamily = 5  // NFPROTO_NETDEV
+	ProtoBridge NFProtoFamily = 7  // NFPROTO_BRIDGE
+	ProtoIPv6   NFProtoFamily = 10 // NFPROTO_IPV6
+	ProtoDECNet NFProtoFamily = 12 // NFPROTO_DECNET
+)
+
+type NFProtoFamily uint8
```

Comment:
I'd probably skip the types an constants in favour of https://pkg.go.dev/golang.org/x/sys/unix#NF_INET_FORWARD etc. Maintaining these manually doesn't add a lot of value. WDYT?

Reply:
Protocol Family and hook constants removed as already we have them at `sys/unix`.